ts/a11y/semantic-enrich.ts

```diff
@@ -245,7 +248,11 @@ export function EnrichedMathDocumentMixin<N, T, D, B extends MathDocumentConstru
         ...BaseDocument.OPTIONS.renderActions,
         enrich:       [STATE.ENRICHED],
         attachSpeech: [STATE.ATTACHSPEECH]
-      })
+      }),
+      a11y: {
```

Comment:
I think you want this to be `expandable()` as well.  What that does is prevents the error checking when options are given in the configuration, so that the options for the explorer will not cause an error if they are included in the configuration but the explorer isn't loaded yet.  In fact, we may need to add an expandable `a11y` option in one of the parent classes so that even if `semantic-enrich` is not loaded, the `a11y` options can be included in the configuration without error.

This does mean, however, that there will be no errors reported for misspelled options, for example.  it might be worth removing `expandable()` from the explorer options, since that is at the top level.  I think the MathDocument is recreated when explorer is loaded, and that would cause the error checking to occur at that point, at least.